lib/LaTeXML/Core/Gullet.pm

```diff
@@ -396,12 +387,6 @@ sub readBalanced {
     Error('expected', "}", $self, "Gullet->readBalanced ran out of input in an unbalanced state.",
       $loc_message); }
   ## Performance enhancement:
```

Comment:
I got a chuckle from noticing that you left the "performance enhancement" part of the comment in, would be great if such comments enhanced performance on their own :>